### ISSUE TITLE
django things

### ISSUE THREAD

**Issue by ajgrover** [zsaraf/weeklyvibes#1]
for venue: https://docs.djangoproject.com/ja/1.9/ref/contrib/gis/model-api/#pointfield

for song: https://docs.djangoproject.com/en/1.9/ref/models/fields/#filefield

**Comment by zsaraf** [comment]
do you use a file field even if the file is stored on s3? or is that only for locally stored files?

**Comment by zsaraf** [comment]
but yes, PointField is 👌🏾

**Comment by ajgrover** [comment]
yeah, you can set up S3 as a storage backend. then it will handle all uploading and stuff for you. https://django-storages.readthedocs.org/en/latest/backends/amazon-S3.html (we use the both-based one  i think)

**Comment by ajgrover** [comment]
you should check out more of the django geo framework, you can do really nice things like `point__within=<geometry>` and `point__distance_lte=point2, distance`. not sure if there's an application for that for this project since everything can be based on a single region but it might let you do more complex searches and order based on distance and stuff
